### DISCUSSION TITLE
Practica Html Pasteleria, ya lo pueden visualizar!

### DISCUSSION THREAD

**Discussion by Will0220** [Launch-X-Latam/MisionFrontEnd#257] [General]
Hola chicos les dejo algo que he adelantado de la practica de la pasteleria!! 

Acepto recomendaciones y sus puntos de vista!! Gracias chicos!

https://will0220.github.io/pasteleria.github.io/

**Comment by floressdi** [comment]
Te quedo muy bien!!!

**Comment by LauraHenao27** [comment]
![Screenshot_2022-10-27-16-37-35-184_com.android.chrome.jpg](https://user-images.githubusercontent.com/114243128/198403228-58ca8380-7b11-49db-9771-ad50e66e111e.jpg)

Hola, teniendo en cuenta el @media, revísale el responsive para celular

**Reply by Will0220** [reply]
Muchas gracias, si lo estoy revisando :)

**Comment by maxkario** [comment]
Hola, esta muy bueno. Me diste una idea para lograr mi practica. Por lo demas me encanto. 
Por otra parte, como un usuario cualquiera que podria ser yo al visitar tu pagina web, el color de fondo es lo que choca un poco. 
Podrias utilizar el header con un color blanco o un gris muy claro. Para el resto del body puedes utilizar un degradado con colores pasteles (te dejare un enlace para degradados al final) para que sea menos chocante. 
Otra cosa, utiliza un H1 solo una vez y recomendablemente en el header. Puedes seccionar lo que dices en el header entre titulo y subtitulo (espero me entiendas) y en el footer colocar algo referente a la empresa. 
por todo lo demas, como mencionas, tienes que mejorar las @medias. Saludos espero que te sirva de ayuda! 

https://webgradients.com/

**Reply by Will0220** [reply]
Muchas gracias !!!

**Comment by OscarJG20** [comment]
Hola!
Como bien lo mencionan los compañeros, ten cuidado con el responsive, dejando de lado que los elementos se pueden acomodar, esa barra horizontal es la primera mala señal. Eso quiere decir que tienes contenido desbordado (a veces es un dolor de cabeza encontrar el motivo).
![image](https://user-images.githubusercontent.com/64516675/198696895-bcf321b8-54d8-4b6d-939b-0a484b7eff0e.png)

Te explico porque sucede en tu caso, tienes un header con un width de 100%, pero en ese mismo elemento también tienes un padding con 10px a los lados. Es como si tuvieras una botella de agua de 500ml, pero tú le quieres agregar 10 ml más, lo que pasará es que el agua ya no entrará, se quedará en el piso.

**Reply by Will0220** [reply]
Gracias, lo tendre en cuenta para mejorarlo 💯
